standard.go

```diff
@@ -0,0 +1,148 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/google/go-querystring/query"
+)
+
+const (
+	standardPath = "/standards"
+)
+
+// Standard defines a PagerDuty's resource standard.
```

Comment:
nit: There should not be an apostrophe here: _defines a PagerDuty resource standard_